src/includes/getting-started-primer/java.mdx

```diff
@@ -0,0 +1,13 @@
+<Note>
+<markdown>
+
+_Sentry's Java SDK enables reporting messages and errors._
```

Comment:
This kind of leaves out Sessions (release health) and Performance (which indeed we lack but it's landing in a few weeks).
What about we just leave this line out altogether?

Reply:
Let's add Sessions (release health) - we an add performance when we update the page to show how to set that up in Java, since we'll need to update the code sample at that time.